metatron-app/metatron-discovery
# #2592 Contents in Column List of Metadata Management and Exploration Data Do Not Match

fix explore metadata detail column tab's type column

### Description

Fix Explore metadata details's column tab's type column display error

**Related Issue** : 

#2592 

### How Has This Been Tested?

1. Exploration -> Data -> click any data 
2. click columns tab
3. check type column show proper data
#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
